execution_engine_testing/test_support/src/wasm_test_builder.rs

```diff
@@ -86,25 +86,25 @@ pub type LmdbWasmTestBuilder = WasmTestBuilder<DataAccessLayer<LmdbGlobalState>>
 
 /// Builder for simple WASM test
 pub struct WasmTestBuilder<S> {
-    /// [`EngineState`] is wrapped in [`Rc`] to work around a missing [`Clone`] implementation
+    /// [`EngineState`] is wrapped in [`Rc`] to work around a missing [`Clone`] implementation.
     engine_state: Rc<EngineState<S>>,
-    /// [`ExecutionResult`] is wrapped in [`Rc`] to work around a missing [`Clone`] implementation
+    /// [`ExecutionResult`] is wrapped in [`Rc`] to work around a missing [`Clone`] implementation.
     exec_results: Vec<Vec<Rc<ExecutionResult>>>,
     upgrade_results: Vec<Result<UpgradeSuccess, engine_state::Error>>,
     prune_results: Vec<Result<PruneResult, engine_state::Error>>,
     genesis_hash: Option<Digest>,
     /// Post state hash.
     post_state_hash: Option<Digest>,
-    /// Cached transform maps after subsequent successful runs i.e. `transforms[0]` is for first
-    /// exec call etc.
-    transforms: Vec<ExecutionJournal>,
-    /// Cached genesis transforms
+    /// Cached effects after successful runs i.e. `effects[0]` is the collection of effects for
+    /// first exec call, etc.
+    effects: Vec<ExecutionJournal>,
```

Comment:
How about

```suggestion
    journal: Vec<ExecutionJournal>,
```
also the comment mentions effects rather than journal. I see you're naming a vec of ExecJournal "effects"?

Reply:
I was aiming to consistently stick with calling this `effects` as that's the term people are most familiar with.  I'd actually like to go the whole way and rename `ExecutionJournal` to `Effects`, but I don't know if that would be acceptable to others?

Reply:
im ok with it